src/App.tsx

```diff
@@ -3,55 +3,83 @@
  * https://github.com/facebook/react-native
  * @flow
  */
-
 import * as React from 'react';
 import {
-  Platform,
-  StyleSheet,
-  Text,
-  View
+    Platform,
+    StyleSheet,
+    Text,
+    View,
+    ToolbarAndroid
 } from 'react-native';
+import Toolbar, {
+    LeftButtonTypes,
+} from './Components/Toolbar'
+import icons, {
+    images
+} from './assets/index';
+
 
 const instructions = Platform.select({
-  ios: 'Press Cmd+R to reload,\n' +
-    'Cmd+D or shake for dev menu',
-  android: 'Double tap R on your keyboard to reload,\n' +
-    'Shake or press menu button for dev menu',
+    ios: 'Press Cmd+R to reload,\n' +
+        'Cmd+D or shake for dev menu',
+    android: 'Double tap R on your keyboard to reload,\n' +
+        'Shake or press menu button for dev menu',
 });
 
-export default class App extends React.Component<{}, {}> {
-  render() {
-    return (
-      <View style={styles.container}>
-        <Text style={styles.welcome}>
-          Hello! This is the agritrader app!!
-        </Text>
-        <Text style={styles.instructions}>
-          To get started, edit App.js
-        </Text>
-        <Text style={styles.instructions}>
-          {instructions}
-        </Text>
-      </View>
-    );
-  }
+export default class App extends React.Component < {}, {} > {
+    render() {
+        return ( <
+            Toolbar title = 'Agritrader'
+            /*as a demo I have placed two buttons on the right
+            one with a settings icon and another with an account icon*/
+            rightButtons = {
```

Comment:
I think for now we don't need to merge in this demo to show it works. Everything we merge should technically be production ready! :) It is great though that this shows that it works
Instead, we can write unit tests to verify it's functionality. I'll send some links for how to create unit tests and I can also help you out with it!

Reply:
What do you think about me removing all of the toolbar related items apart for the imports.